packages/components/src/color-edit/index.js

```diff
@@ -213,24 +227,26 @@ export default function ColorEdit( { colors = EMPTY_ARRAY, onChange } ) {
 							{ __( 'Done' ) }
 						</DoneButton>
 					) }
-					<Button
-						isSmall
-						isPressed={ isAdding }
-						icon={ plus }
-						label={ __( 'Add custom color' ) }
-						onClick={ () => {
-							onChange( [
-								...colors,
-								{
-									color: '#000',
-									name: '',
-									slug: '',
-								},
-							] );
-							setIsEditing( true );
-							setEditingColor( colors.length );
-						} }
-					/>
+					{ ! canOnlyChangeValues && (
+						<Button
+							isSmall
+							isPressed={ isAdding }
+							icon={ plus }
+							label={ __( 'Add custom color' ) }
```

Comment:
It sounds like the ColorEdit component should be agnostic as to what's the source of the color. I'd think this label should be provided by the consumer or be called `Add color`.